src/components/Section.js

```diff
@@ -2,7 +2,7 @@ export default class Section {
   constructor({ items, renderer }, containerSelector) {
     this._renderer = renderer;
     this._renderedItems = items;
-    this._container = document.querySelector(containerSelector);
+    this._container = containerSelector;
```

Comment:
Needs correction: Selector is a string, but here we have an element, so rename it to `container`